codecov.yml

```diff
@@ -0,0 +1,2 @@
+comment: false
+threshold: 5%
```

Comment:
just curious - what are these settings for?

Reply:
https://docs.codecov.com/docs/commit-status#branches